<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>D6 client</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="portLabel">
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="portDevice">
          <property name="text">
           <string>/dev/ttyUSB1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="baudRateLabel">
          <property name="text">
           <string>Baud rate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="baudRate"/>
        </item>
        <item>
         <widget class="QPushButton" name="connectButton">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="connStatus">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Not connected</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="firmwareVersionLabel">
          <property name="text">
           <string>Firmware version:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="firmwareVersion">
          <property name="text">
           <string>UNKNOWN</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="vfoLabel">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>VFO Freq.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="vfoFreq">
          <property name="maximum">
           <double>9999.989999999999782</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="vfoUnitsComboBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="setVFOButton">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="scanButton">
          <property name="text">
           <string>Scan</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="continuousScanButton">
          <property name="text">
           <string>Continuous</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QDoubleSpinBox" name="maxLevel">
            <property name="minimum">
             <double>-100.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>max level
(dB)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>min level
(dB)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="minLevel">
            <property name="minimum">
             <double>-100.000000000000000</double>
            </property>
            <property name="value">
             <double>-87.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QChartView" name="fftView"/>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="7">
           <widget class="QDoubleSpinBox" name="endFreq">
            <property name="maximum">
             <double>9999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QDoubleSpinBox" name="spanFreq">
            <property name="maximum">
             <double>9999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Center freq.</string>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Span freq.</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="startFreqComboBox"/>
          </item>
          <item row="2" column="7">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>End freq.</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QDoubleSpinBox" name="centerFreq">
            <property name="maximum">
             <double>9999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Start freq.</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QDoubleSpinBox" name="startFreq">
            <property name="maximum">
             <double>9999.000000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="6">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="stepSizeLabel">
        <property name="text">
         <string>Step size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="stepSize">
        <property name="maximum">
         <double>9999.989999999999782</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="stepSizeComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="samplesLabel">
        <property name="text">
         <string>Samples</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="samples">
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="logCheckBox">
        <property name="text">
         <string>Logarithmic</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>866</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
